### DISCUSSION TITLE
Misidentification

### DISCUSSION THREAD

**Discussion by tomlut** [tphakala/birdnet-go#91] [General]
Is there anything I can do about misidentification?

Birdnet-go is constantly misreporting Yellow Wattlebirds as Ring-Necked Pheasants (confirmed visually).

**Comment by tphakala** [comment]
Currently only options to control BirdNET analyzer model itself is through these birdnet settings

    birdnet:
        sensitivity: 1.0
        threshold: 0.8
        latitude: 61.100
        longitude: 25.100

Tweaking sensitivity setting lower or higher might help but it is trial and error effort which may result in nothing.

Increasing threshold should reduce number of misidentifications but it will often reduce number of correct detections also lowering number of total identifications.

Latitude and Longitude settings are quite important as that will limit species range to what are native at your region, e.g. removing completely incorrect species from results.

However birdnet-go supports species specific threshold setting through config file names species_settings.csv, I should have included example file in latest release bundle but I simply forgot to include it in there, but you can find example file at this URL https://[raw.githubusercontent.com/tphakala/birdnet-go/main/doc/species_config.csv](https://raw.githubusercontent.com/tphakala/birdnet-go/main/doc/species_config.csv)

So in your case I would maybe have entries like this species_config.csv

yellow wattlebird,0.2
ring-necked pheasant,0.99

On Windows species_config.csv should be placed in same directory with birdnet-go.exe, on Linux it goes to ~/.config/birdnet-go/ or to /etc/birdnet-go/

Above settings lowers detection threshold of Yellow Wattlebird to confidence of 20% and increases detection confidence value of Ring-Necked Pheasant to 99%. End result is that Yellow Wattlebirds are reported more often due to lower confidence value allowing them to be reported and Ring-Necked Pheasants require much higher detection confidence to be reported.

Ultimately these misdetections should be corrected at BirdNET model level but I have no control over it as I am just using model provided by https://github.com/kahst/BirdNET-Analyzer project as is.

**Comment by tomlut** [comment]
Yeah my lat & lon are set correctly. 

The specific species config was exactly what I was after (I did check the docs because I was sure I'd seen it before).

 Thank you.

**Comment by tphakala** [comment]
I forgot to mention that you need to restart birdnet-go process to reload config

**Comment by tomlut** [comment]
Yep. Figured that would be required.
